src/test/java/com/fauna/codec/codecs/EnumCodecTest.java

```diff
@@ -0,0 +1,41 @@
+package com.fauna.codec.codecs;
+
+import com.fauna.beans.ClassWithAttributes;
+import com.fauna.beans.ClassWithClientGeneratedIdCollTsAnnotations;
+import com.fauna.beans.ClassWithFaunaIgnore;
+import com.fauna.beans.ClassWithIdCollTsAnnotations;
+import com.fauna.beans.ClassWithInheritanceL2;
+import com.fauna.beans.ClassWithParameterizedFields;
+import com.fauna.beans.ClassWithRefTagCollision;
+import com.fauna.codec.Codec;
+import com.fauna.codec.DefaultCodecProvider;
+import com.fauna.exception.NullDocumentException;
+import com.fauna.types.Module;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
```

Comment:
Surely some of these imports are unused?

Reply:
Hahahahaha, definitely. Reminds me that a linter for things like this in PR will be nice